_posts/2025-04-24-Announcing-Data-Prepper-2.11.md

```diff
@@ -0,0 +1,254 @@
+
+Data Prepper's new `rds` source first exports existing data from Aurora/Amazon RDS tables to OpenSearch indexes and then streams incremental changes from those tables in order to maintain data consistency between the relational database and OpenSearch.
+The `rds` source currently supports the Aurora MySQL, Aurora PostgreSQL, RDS MySQL, and RDS PostgreSQL engines.
+
+The following is an example configuration:
+
+```yaml
+aurora-mysql-pipeline:
+  source:
+    rds:
+      db_identifier: "my-aurora-cluster"
+      engine: "aurora-mysql"
+      database: "hr_db"
+      tables:
+        include:
+          - "employees"
+          - "departments"
+      s3_bucket: "my-s3-bucket"
+      s3_prefix: "pipeline-data"
+      export:
+        kms_key_id: "1234abcd-1234-abcd-1234-123456abcdef"
+        export_role_arn: "arn:aws:iam::123456789012:role/ExportRole"
+      stream: true
+      aws:
+        sts_role_arn: "arn:aws:iam::123456789012:role/PipelineRole"
+        region: "us-east-1"
+      authentication:
+        username: ${{aws_secrets:secret:username}}
+        password: ${{aws_secrets:secret:password}}
+```
+
+## Amazon SQS as a source
+
+Data Prepper now supports a new Amazon Simple Queue Service (Amazon SQS) source for reading events from SQS queues.
+Amazon SQS is a fully managed message queue.
+The new `sqs` source in Data Prepper efficiently receives messages from Amazon SQS to create events that you can route to sinks.
+
+Data Prepper receives SQS messages from an SQS queue in batches and then creates Data Prepper events from those SQS messages.
+By default, Data Prepper will create a single Data Prepper event per SQS message.
+Data Prepper provides a robust collection of [processors](https://docs.opensearch.org/docs/latest/data-prepper/pipelines/configuration/processors/processors/) that you can use to parse or grok, depending on the format of your message.
+
+To help reduce your Amazon SQS costs, you can instead design your SQS data and configure Data Prepper to support multiple events per message.
+Using this approach reduces your Amazon SQS costs by combining data into SQS messages, allowing for fewer SQS sends and receives.
+To take this approach, you must design your sending application to send SQS messages to Amazon SQS in a format available as a [Data Prepper codec](https://docs.opensearch.org/docs/latest/data-prepper/pipelines/configuration/sources/s3/#codec).
+Then you can configure your Data Prepper pipeline to use that codec to parse the message into multiple events.
+
+## Other features and improvements
+
+This release also includes several additional features to help you create pipelines:
+
+* The `rename_keys` processor can now rename keys with variable names using a regex pattern.
+* The `opensearch` sink now supports new index types for OTel logs and metrics.
+* Data Prepper expressions now support slashes (`/`) in the key by allowing you to escape the `/` character.
```

Comment:
@natebower , I changed this line again to try to add clarity. Please take another look.

Reply:
@dlvenable LGTM